test/System.Web.Mvc.Test/Html/Test/DefaultEditorTemplatesTest.cs

```diff
@@ -731,14 +731,63 @@ public void ObjectTemplateDisplaysNullDisplayTextWithNullModelAndTemplateDepthGr
         public void PasswordTemplateTests()
         {
             Assert.Equal(
-                "<input class=\"text-box single-line password\" id=\"FieldPrefix\" name=\"FieldPrefix\" type=\"password\" value=\"Value\" />",
+                "<input class=\"text-box single-line password\" id=\"FieldPrefix\" name=\"FieldPrefix\" type=\"password\" />",
```

Comment:
This test (well, two tests in one) and the two theories using the data set below set `Model` and `FormattedModelValue`. I left similar settings in `PasswordTemplate_ReturnsInputElement_IgnoresValues()` due to its narrower intent.